website/src/components/events/event-list-item.js

```diff
@@ -85,25 +85,37 @@ export default function EventListItem(props) {
     eventType = "Community Event"
   }
 
+  // Gatsby seems to hijack all <a> tags and, if they do not begin with http://
+  // or https://, appends the path to the domain instead of setting the URL as
+  // expected.  
+  let link = info.link
+  if (!link) link = info.where
```

Comment:
In my local testing of this logic, the event list item would render the 'Where' field as a link if the post included a link to something like an external registration page. Can we keep these separate for now? 

Only if the `info.Where` property contains a potential URL should it be rendered as a link.

Reply:
I have changed this with `info.where` only.